crypto/bn/build.info

```diff
@@ -101,12 +101,18 @@ IF[{- !$disabled{asm} -}]
   ENDIF
 ENDIF
 
+IF[{- !$disabled{dh} -}]
```

Comment:
I assume this bit wasnt actually causing a problem, and you are just stripping constants out becuase they are not needed if DH is not used.
I wonder if you could use them for DSA (they are just p, q, g values).

Reply:
It was causing a problem. The functions in bn_const.c were being compiled unconditionally, but the constants that they referred to (which are in another file) were compiled out if no-dh was specified. Therefore we were ending up with linker errors. I chose to not compile the functions in the event of no-dh. An alternative solution would be to keep the constants compiled in, even in a no-dh build. However, I checked the RFCs and they state that they are intended for use in Diffie Hellman.